package.json

```diff
@@ -14,6 +14,7 @@
     "next": "13.1.1",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "styled-jsx": "5.1.2",
```

Comment:
we are not using this library

Reply:
removed. I don't know why this dependency appear. so weird